src/containers/FeedContainer.ts

```diff
@@ -1,29 +1,44 @@
 import { connect } from 'react-redux';
-import { AppState } from '../reducers';
+import { AppState, defaultLocalPosts } from '../reducers';
 import { StateProps, DispatchProps, FeedView } from '../components/FeedView';
 import { AsyncActions, Actions } from '../actions/Actions';
 import { Feed } from '../models/Feed';
 import { getFeedPosts, getYourPosts } from '../selectors/selectors';
 
 export const mapStateToProps = (state: AppState, ownProps: { navigation: any }): StateProps => {
     const feedUrl = ownProps.navigation.state.params.feedUrl;
-    const selectedFeeds = state.feeds.filter(feed => feed != null && feed.feedUrl === feedUrl);
-    // TODO: we should have here state.ownFeeds.find(feed => feed.url === feedUrl) != null,
-    // but author.uri is empty, and the feedUrl from navparams is from post.author.uri (not real feedUrl)
-    // For the same reason feeds/selectedFeeds is empty (errounously)
-    const isOwnFeed = state.author.uri === feedUrl;
+    const feedName = ownProps.navigation.state.params.name;
```

Comment:
params.name could be more specific

Reply:
I didn't want to change this, because passing params through the navigation state is not type safe and I didn't want to introduce other breaking changes.